dissect/target/loaders/cpio.py

```diff
@@ -0,0 +1,28 @@
+from pathlib import Path
+
+from dissect.util import cpio
+
+from dissect.target import Target
+from dissect.target.filesystems.tar import TarFilesystem
+from dissect.target.helpers.fsutil import open_decompress
+from dissect.target.loader import Loader
+
+
+class CpioLoader(Loader):
```

Comment:
I was thinking about this a little bit more and it might be better to implement this as a `CpioFilesystem` which would be a subclass of `TarFilesystem`. That way it would get picked up automatically by the `RawLoader`, and it's a little more flexible in other use-cases.

Reply:
Sounds like a good improvement for the next PR ;)